api/client.go

```diff
@@ -72,14 +72,16 @@ func (c *Client) Do(req *http.Request) (*http.Response, error) {
 	return res, nil
 }
 
-// ValidateToken calls the API to determine whether the token is valid.
-func (c *Client) ValidateToken() error {
+// TokenIsValid calls the API to determine whether the token is valid.
+func (c *Client) TokenIsValid() (bool, error) {
 	url := fmt.Sprintf("%s/validate_token", c.APIBaseURL)
```

Comment:
This actually might be a good place to use `path.Join(c.APIBaseURL, validate_token)` :laughing:

Reply:
Hah! Now the distinction between `path` and `filepath` is starting to make sense.

Reply:
with path.Join it does this: (removes the // after the protocol)
```
http:/127.0.0.1:61851/validate_token
```

Reply:
:man_facepalming: so much for this statement 
> The path package should only be used for paths separated by forward slashes, such as the paths in URLs.

Sorry for that suggestion without actually testing.